### ISSUE TITLE
 ValueError('Expected 2 or more dimensions (got 1)',), <traceback object at 0x7f04e0d0d288>)

### ISSUE THREAD

**Issue by yjyGo** [GBLin5566/An-Automated-Traditional-Chinese-Dialogue-Generating-System#1]
Can you help me? I can't solve this problem.

/home/yangjy/PycharmProjects/An-Automated-Traditional-Chinese-Dialogue-Generating-System-master/model.py:99: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  output = F.log_softmax(self.out(output[0]))
(<class 'ValueError'>, ValueError('Expected 2 or more dimensions (got 1)',), <traceback object at 0x7f04e0d0d288>)
Get stopped, saving the latest model

**Comment by GBLin5566** [comment]
Hi @yangjingyingGo , what's your Pytorch version ?
